src/plugins/auto/plugin.cpp

```diff
@@ -396,11 +416,8 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
                              config.first.c_str(), config.second.c_str());
                  }
              }
-             auto tmpiter = std::find_if(fullConfig.begin(), fullConfig.end(), [](const std::pair<std::string, std::string>& config) {
-                            return (config.first == CONFIG_KEY(ALLOW_AUTO_BATCHING));
-                            });
-             if (tmpiter != fullConfig.end())
-                 deviceConfig.insert({tmpiter->first, tmpiter->second});
+             insertPropToConfig(CONFIG_KEY(ALLOW_AUTO_BATCHING), iter->deviceName, deviceConfig);
+             insertPropToConfig(CONFIG_KEY(AUTO_BATCH_TIMEOUT), iter->deviceName, deviceConfig);
```

Comment:
BTW, why AUTO must know about auto batch properties?

Reply:
Here auto just act as a relay to avoid the auto batch property missed before it was correctly handled in ie core(allow_auto_batch) or batch plugin(batch_timeout).  for example, ie.compile_model("AUTO", ov::hint::allow_auto_batching(NO), if auto does not handle this, ie core will not have this information to decide if batch should be disabled when load network to hw plugin in auto.